### PR TITLE
feat(ospm): create new page

### PR DESCRIPTION
**Pull request by lavanya-bmw**
## Description

Create new page for onboarding service provider management

## Why

NA

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/541

## Checklist

Please delete options that are not relevant.

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

File: src/types/Constants.ts [hunk 1]
```diff
@@ -110,6 +110,7 @@ export enum PAGES {
   COMPANY_SUBSCRIPTIONS = 'company-subscriptions',
   COMPANY_SUBSCRIPTIONS_DETAIL = 'company-subscriptions-detail',
   COMPANY_DATA = 'company_data',
+  ONBOARDING_SERVICE_PROVIDER = 'onboardingServiceProvider',
```
Comment (oyo) [inline src/types/Constants.ts:113]: Above in Line 105 we already have 

`ONBOARDING_SERVICEPROVIDER = 'onboarding-serviceprovider',`

Is this a duplicate?

File: src/components/pages/OnboardingServiceProvider/OnboardingServiceProvider.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+              sx={{
+                right: '0',
+                position: 'absolute',
+                top: '50%',
+                msTransform: 'translateY(-50%)',
+                transform: 'translateY(-50%)',
+              }}
+            >
+              <ArrowForwardIcon />
+            </IconButton>
```
Comment (oyo) [inline src/components/pages/OnboardingServiceProvider/OnboardingServiceProvider.tsx:120]: this block is very similar to the one in L87-L99 - I suggest you create a local component on top of the file and just specify the child and right / left

File: src/components/pages/OnboardingServiceProvider/OnboardingServiceProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+              textTransform: 'none',
+              display: 'inline-flex',
+              width: '100%',
+              maxWidth: '550px',
+              '&.Mui-selected': {
+                borderBottom: '3px solid #0f71cb',
+              },
+            }}
+          />
+        </Tabs>
```
Comment (oyo) [inline src/components/pages/OnboardingServiceProvider/OnboardingServiceProvider.tsx:163]: again too much duplication - better create a local component with props for the differences

**Review by oyo — CHANGES_REQUESTED** [review]
a few suggestions and one question

**Review by evegufy — CHANGES_REQUESTED** [review]
testing for 24.08 ends officially in less than 24h, please don't merge without me confirming but the change will very likely not go into the 2.1.0 release anymore, I'll let you know tomorrow

**Comment by evegufy** [comment]
confirmed: v2.1.0 is closed,  please change to main

**Comment by MaximilianHauer** [comment]
@lavanya-bmw  can you please have a look on the notes of Martin.

**Pull request closed without merging**